source/_integrations/cover.markdown

```diff
@@ -35,15 +35,15 @@ Available services: `cover.open_cover`, `cover.close_cover`, `cover.stop_cover`,
 
 | Service data attribute | Optional | Description |
 | ---------------------- | -------- | ----------- |
-| `entity_id` | yes | String or list of strings that point at `entity_id`'s of covers. Else targets all.
+| `entity_id` | yes | String or list of strings that point at `entity_id`'s of covers. Either `entity_id` or `area` must be specified. Use `entity_id: all` to target all.
```

Comment:
The area is a different attribute...
So that should not be in here.

Reply:
Okay, how would you suggest to make it clear that either an entity_id or area must be provided? If I just leave this part out, they are both optional.
Actually, area isn't mentioned yet. Shall I put that parameter in the table for every integration?

Reply:
`area_id` is the additional parameter allowed.

However, it is not clear on how to use that at this point, and is something we need to look into.

It does, however, show up, if you create an automation with the automation editor. Create a automation targeting an area, and you'll see it in the resulting generated YAML.